utils/constants.js

```diff
@@ -0,0 +1,30 @@
+  // You can use string format or `process.env` if you'd prefer
+const uuidv4 = require(`uuid/v4`);
+const pluginOptions = require(`../../../gatsby-config`);
+const ci = require('ci-info');
+
+let THEME_OPTIONS = pluginOptions.plugins.filter(plugin => plugin.resolve === 'gatsby-plugin-newrelic-test')[0].options;
```

Comment:
If we need to adjust the structure of the plugin options, we would need to update the config, the place that reads the options, _and_ this location. I suggest that we supply the fallback values at the time that we need to read them (I'll comment there when possible).

Unrelated tip: when you are looking for a specific item in an array, rather than using `filter` and `[0]` you might want to reach for `find` instead:

```js
const options = pluginOptions.plugins
  .find(plugin => plugin.resolve === 'gatsby-plugin-newrelic-test').options;
```

This could be further cleaned up (just a matter of style preference tho):

```js
const PLUGIN_NAME = "gatsby-plugin-newrelic-test";
const DEFAULT_OPTIONS = { /* stuff */ };

const { plugins } = pluginOptions;
const plugin = plugins.find(({ resolve }) => resolve === PLUGIN_NAME);
const options = plugin ? plugin.options : DEFAULT_OPTIONS;
```

Reply:
very related tip 😻 I think that definitely looks cleaner and makes more sense from a chore perspective, good call